state-sync/state-sync-v2/state-sync-driver/src/driver.rs

```diff
@@ -285,6 +290,10 @@ impl<
             sync_notification.target, latest_synced_version,
             ))
         );
+        metrics::increment_counter(
+            &metrics::DRIVER_COUNTERS,
+            metrics::DRIVER_CONSENSUS_SYNC_NOTIFICATION.into(),
+        );
```

Comment:
Why are these better than just calling `with_label_values()` and `inc()`? 

And we should not need to allocate the `String`s.

Reply:
I agree, take in an `&str`

Reply:
> Why are these better than just calling with_label_values() and inc()?

It's mostly a style-choice and it saves lines (e.g., the wrapping often results in more space per invocation).

> And we should not need to allocate the Strings.

Unfortunately, this is mostly copy/paste. I've fixed it in this PR, but there's others that will eventually need attention 😄

Reply:
thank you, have a nice day